x-pack/plugins/security_solution/public/common/store/store.ts

```diff
@@ -272,23 +266,18 @@ export const createStore = (
 
   const composeEnhancers = composeWithDevTools(enhancerOptions);
 
-  const middlewareDependencies: TimelineEpicDependencies<State> = {
-    kibana$: kibana,
-    selectAllTimelineQuery: inputsSelectors.globalQueryByIdSelector,
-    selectNotesByIdSelector: appSelectors.selectNotesByIdSelector,
-    timelineByIdSelector: timelineSelectors.timelineByIdSelector,
-    timelineTimeRangeSelector: inputsSelectors.timelineTimeRangeSelector,
-  };
-
-  const epicMiddleware = createEpicMiddleware<Action, Action, State, typeof middlewareDependencies>(
-    {
-      dependencies: middlewareDependencies,
-    }
-  );
+  // TODO: Once `createStore` does not use redux-observable, we will not need to pass a
+  // kibana observable anymore. Then we can remove this `any` cast and replace kibana$
+  // with a regular kibana instance.
+  // I'm not doing it in this PR, as this will have an impact on literally hundreds of test files.
+  // A separate PR will be created to clean this up.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const kibanaObsv = kibana$ as any;
+  const kibana =
+    'source' in kibanaObsv ? kibanaObsv.source._value.kibana : kibanaObsv._value.kibana;
```

Comment:
See comment in code. If we'd change `kibana` from an observable now, we'd have tons of test files that need updating alongside this PR. Given, `kibana` is a constant, it's "safe" to access it through the observable internals.